worker/provision_external_device.json

```diff
@@ -52,16 +52,39 @@
                     404
                 ]
             }
+        },
+        {
+            "name": "create_configuration_deployment",
```

Comment:
isn't the deviceconfing, called in a previous step, creating configuration deployment?

Reply:
No, the call to deviceconfig only updates it's internal representation of the desired configuration.

Reply:
However, this reminds me, I should probably use the deviceconfig `/deploy` endpoint instead of calling deployments directly to avoid any potential data race condition.